src/+mpecopt/Solver.m

```diff
@@ -781,31 +782,45 @@ function process_solver_initialization(obj, solver_initialization)
             end
 
             if opts.lift_complementarities_full
```

Comment:
lot of code in the if and else is the same, put it outside the if and else to have fewr lines.